src/Storages/ObjectStorageQueue/StorageObjectStorageQueue.cpp

```diff
@@ -210,15 +210,16 @@ StorageObjectStorageQueue::StorageObjectStorageQueue(
     , reschedule_processing_interval_ms((*queue_settings_)[ObjectStorageQueueSetting::polling_min_timeout_ms])
     , log(getLogger(fmt::format("Storage{}Queue ({})", configuration->getEngineName(), table_id_.getFullTableName())))
 {
-    if (configuration->getPath().empty())
+    const auto raw_path = configuration->getRawPath();
+    if (raw_path.path.empty())
```

Comment:
Caching path for read kind of conflicts with this: https://github.com/ClickHouse/ClickHouse/pull/76802/commits/5d0aa96041a6339f910569713706ea8926b8d1a0.

Might need to re-think or drop the "cache"

Reply:
Reverting the 'cache' for now